lambda/src/redirect.js

```diff
@@ -0,0 +1,32 @@
+const streamClient = require('./streamClient');
+
+const sendClickEvent = (outboundUrl) => {
+  return streamClient.publish('link-clicked', {
+    outboundUrl: outboundUrl
+  }).catch(() => Promise.reject({status: 500, message: 'Failed to publish event'}));
+};
+
+const redirect = (queryStringParameters) => {
+  if (queryStringParameters && queryStringParameters['outboundUrl']) {
+    sendClickEvent(queryStringParameters['outboundUrl'])
+      .catch((e) => console.log(e));
```

Comment:
I think we have some redundant promise `catch`-ing going on here. On line 6, we catch a rejection from `streamClient.publish`, and re-throw it as a very generic failure thing. Then here on line 12 we catch the generic failure object and log it. But there won't be any useful diagnostic information to log, because it was all thrown away on line 6.

Anyway I think we can just remove the catch from line 6, then let this code here on line 12 do the catching and logging. Or alternatively line 6 could be changed to do the logging, but not re-reject, and then line 12 can go away.

Also the catch-and-log can be done like this, without the extra arrow function: `.catch(console.error);`.